eth2deposit/deposit.py

```diff
@@ -1,22 +1,54 @@
-import sys
 import click
+import sys
 
 from eth2deposit.cli.existing_mnemonic import existing_mnemonic
 from eth2deposit.cli.new_mnemonic import new_mnemonic
+from eth2deposit.utils.click import (
+    captive_prompt_callback,
+    choice_prompt_func,
+    jit_option,
+)
+from eth2deposit.utils import config
+from eth2deposit.utils.constants import INTL_LANG_OPTIONS
+from eth2deposit.utils.intl import (
+    get_first_options,
+    fuzzy_reverse_dict_lookup,
+    load_text,
+)
 
 
 def check_python_version() -> None:
     '''
     Checks that the python version running is sufficient and exits if not.
     '''
     if sys.version_info < (3, 7):
-        click.pause('Your python version is insufficient, please install version 3.7 or greater.')
+        click.pause(load_text(['err_python_version']))
         sys.exit()
 
 
 @click.group()
-def cli() -> None:
-    pass
+@click.pass_context
+@jit_option(
+    '--language',
+    callback=captive_prompt_callback(
+        lambda language: fuzzy_reverse_dict_lookup(language, INTL_LANG_OPTIONS),
+        choice_prompt_func(lambda: 'Please choose your language', get_first_options(INTL_LANG_OPTIONS)),
+    ),
+    default='English',
+    help='The language you wish to use the CLI in.',
+    prompt=choice_prompt_func(lambda: 'Please choose your language', get_first_options(INTL_LANG_OPTIONS))(),
+    type=str,
+)
+@click.option(
+    '--non_interactive',
+    default=False,
+    is_flag=True,
+    help='Disables interactive prompts.',
```

Comment:
Unimportant, just FYI it didn't show when I execute `python eth2deposit/deposit.py  --help`. I guess it's because deposit CLI mixes `click.option` and custom `jit_option`?

Reply:
It is because I purposefully hid this option, I intended only to use it for testing. (See [L47](https://github.com/ethereum/eth2.0-deposit-cli/pull/182/files/6f388d842ab32e239690ee56304e90952491edaa#diff-dfb2523fb295de650586cb72981df1988c32ef0749034028f84426db0a2da10aR47) for hidden argument.)

Are you of the opinion that we should be opening this up to users? (It may be useful to those integrating this into other tools)